src/eventshub/vite.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  build: {
+    rollupOptions: {
+      output: {
+        entryFileNames: 'eventshub.js',
+        chunkFileNames: 'eventshub-chunk.js',
+        assetFileNames: '[name].[ext]',
```

Comment:
We don't have a build system on the sites stack as far as I recall so don't we need to track the dist dir as well?

Reply:
Ah yes! That we do 😄 Let me update the `.gitignore`